### ISSUE TITLE
Edit checklist in Preview

### ISSUE THREAD

**Issue by Fakh94** [uiwjs/react-md-editor#491]
Is it possible to check a checkbox option in the preview mode ?

**Comment by jaywcjlove** [comment]
It is feasible, but you need to define a plugin using the `previewOptions` props to change the attributes of a checkbox, and then listen to the input to change the document text.

**Comment by Fakh94** [comment]
Thank you but do you have an idea which plugin should i use to do that

**Comment by Fakh94** [comment]
> It is feasible, but you need to define a plugin using the `previewOptions` props to change the attributes of a checkbox, and then listen to the input to change the document text.

Yes, i think that's possible through a plugin, but do you have an idea which plugin should i use
